rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "stable"
```

Comment:
Originally, I had removed the `toolchain.toml` (with unstable channel), since we now work on stable. This toml will override the user's default toolchain (ex. if they set their default to nightly) . I think re-addition w/ stable is fine since people can still [override with the +nightly shorthand, RUSTUP_TOOLCHAIN env var, or directory override](https://rust-lang.github.io/rustup/overrides.html#overrides)

Reply:
I like being explicit and I think this will help catch nightly things to make sure nightly features are used intentionally